### PR TITLE
docs: updated contributing.md and readme.md

### PR DESCRIPTION
**Pull request by tbarn**
## Checklist

- [ ] Tests have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## What kind of change does this PR introduce

Docs update

## What is the current behavior

Longer readme, where the last sections are mostly only needed if contributing 

## Does this PR introduce a breaking change

Nope 

### Other information

Before: 
Screen Shot 2019-05-09 at 4 32 31 PM

After: 
Screen Shot 2019-05-09 at 4 35 59 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,45 +119,25 @@ If you have a base path of `/api` and your path is defined as `hello`, then a re
...
-## Debugging
-
-The best way to debug a Prism behaviour is probably to attach your debugger to the CLI and go from there. To make that happen:
-
-```bash
-cd packages/cli
-
-node --inspect-brk -r tsconfig-paths/register bin/run mock file.oas.yml
-```
+**The API description defines a base path of `/api` (using OpenAPI v2 `basePath` keyword, or in
```
Comment (philsturgeon) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -132,45 +119,25 @@ If you have a base path of `/api` and your path is defined as `hello`, then a re
...
+OpenAPI v3 using a path in `servers.url`), but requests seem to fail when using it?**
```
Comment (philsturgeon) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -126,45 +119,22 @@ If you have a base path of `/api` and your path is defined as `hello`, then a re
...
-Prism has an extensive test suite. To run it, just use the regular `test` script
-
-```bash
-yarn test
-# or
-npm test
-```
+- [ ] Support files ending with `.yml` and extension-less files
 
-## Debugging
```
Comment (XVincentX) [inline README.md]: I think the debugging section should stay or at least put a link in the relevant contributing section. Last place where I would look for technical help is the contributing file, what do you think @philsturgeon @tbarn

Comment (tbarn) [reply README.md]: I can add a link in another PR today.

**Pull request merged**